### PR TITLE
Add File Overlay Spinner

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add new custom overlay spinner for use specifically during file uploads

### CODE REVIEW

File: portal-frontend/src/app/shared/overlay-spinner/upload-spinner.component.ts
Comment (sandratoh) [inline portal-frontend/src/app/shared/overlay-spinner/upload-spinner.component.ts:1]: Can we remove this?

**Review by sandratoh — APPROVED** [review]
LGTM 👍

File: portal-frontend/src/app/shared/overlay-spinner/upload-spinner.component.ts
Comment (dhaselhan) [reply portal-frontend/src/app/shared/overlay-spinner/upload-spinner.component.ts:1]: oops, yep! Accidentally made that.

Comment (dhaselhan) [reply portal-frontend/src/app/shared/overlay-spinner/upload-spinner.component.ts:1]: Removed.

**Pull request merged**